src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Oh, yes.  THAT picture.  that really summarizes it nicely.  Sorry guys, we
are stuck with multiple parents.  Biology is complex.

Daniel Zerbino <notifications@github.com> writes:
> In src/main/resources/avro/sequenceAnnotations.avdl:
> 
> >      */
> > -    array<string> childrenIds;
> > +    string parentId;
> 
> I don't know if Github allows attachments, so I'm sending you the same diagrams
> directly.
> …
> On Wed, Feb 4, 2015 at 8:05 AM, Mark Diekhans <notifications@github.com> wrote:
> In src/main/resources/avro/sequenceAnnotations.avdl <https://github.com/
> diekhans/ga4gh-schemas-genome-ann/pull/16#discussion_r24067930> : > */ > -
> array<string> childrenIds; > + string parentId; Daniel Zerbino
> <notifications@github.com> writes: Each object has a parent, but GFF3 also
> allows a derived_from tag, kinda like a secondary ancestor link. That means
> that although Gene1 is parent to Transcript1 and Transcript1 is parent to
> Exon1, you are free to have Exon1 derived from another gene. Also, I am looking
> at the spec again and remembering some of the other uses of multiple parents. I
> need to draw some pictures, but it actually feels like a very elegant solution
> to some ugly problems. That Lincoln is a smart and pragmatic guy... — Reply to
> this email directly or view it on GitHub <https://github.com/diekhans/
> ga4gh-schemas-genome-ann/pull/16/files#r24067930> .
> 
> —
> Reply to this email directly or view it on GitHub.*
>